sites/default/settings.php

```diff
@@ -0,0 +1,243 @@
+    // Name transaction "redirect" in New Relic for improved reporting (optional).
+    if (extension_loaded('newrelic')) {
+      newrelic_name_transaction("redirect");
+    }
+    exit();
+  }
+
+  // Reverse proxy configuration
+  if (isset($_SERVER['REMOTE_ADDR']) && (in_array($_SERVER['REMOTE_ADDR'], $conf['reverse_proxy_addresses'], TRUE))) {
+    // Domain masking:
+    // Some sites on Pantheon share a common domain name (e.g. www.uoguelph.ca). Pantheon
+    // refers to this as 'domain masking' and it is not officially supported. These sites
+    // are forwarded through an Apache reverse proxy server. The original domain name of
+    // the site is translated to the Pantheon platform URL. On the Pantheon side, Drupal
+    // must be tricked into thinking it is running under the primary domain name from a
+    // subdirectory. The general case is to rewrite the request URL to:
+    // www.uoguelph.ca/{sitename}, but there are exceptions.
+    // Every site on Pantheon has a "platform domain" derived from the site name and 
+    // environment, ending in pantheonsite.io.
+    $platform_domain = $env . '-' . $site . '.pantheonsite.io';
+
+    // Site specific reverse proxy configuration. If a reverse proxy site does not
+    // follow the https://www.uoguelph.ca/sitename pattern, override the reverse proxy
+    // domain and path here.
+    $proxy_conf = array(
+      'a11yportal' => array('domain' => 'wellness.uoguelph.ca', 'path' => '/accessibility/portal'),
+      'accessibilityweb' => array('path' => '/accessibility/web'),
+      'ahl1' => array('path' => '/ahl'),
+      'alliance1' => array('path' => '/alliance'),
+      'arts1' => array('path' => '/arts'),
+      'bioinformatics1' => array('path' => '/bioinformatics'),
+      'business2' => array('path' => '/lang'),
+      'cbspurchasing' => array('path' => '/cbs/purchasing'),
+      'ccs1' => array('path' => '/ccs'),
+      'ccsforms' => array('path' => '/ccs/forms'),
+      'cecs' => array('domain' => 'www.recruitguelph.ca'),
+      'cio1' => array('path' => '/cio'),
+      'cip1' => array('path' => '/cip'),
+      'counselling' => array('domain' => 'wellness.uoguelph.ca'),
+      'cps1' => array('path' => '/cps'),
+      'dhr1' => array('path' => '/diversity-human-rights'),
+      'dunfield' => array('path' => '/crc/dunfield'),
+      'engineering1' => array('path' => '/engineering'),
+      'finance1' => array('path' => '/finance'),
+      'fire1' => array('path' => '/fire'),
+      'fsadmin' => array('path' => '/foodscience/fsadmin'),
+      'fys' => array('path' => '/vpacademic/fys'),
+      'graduatestudies' => array('domain' => 'graduatestudies.uoguelph.ca'),
+      'graduatestudiesforms' => array('path' => '/graduatestudies/forms2'),
+      'gsa1' => array('path' => '/gsa'),
+      'healthservices' => array('domain' => 'wellness.uoguelph.ca', 'path' => '/health'),
+      'hpc1' => array('domain' => 'wellness.uoguelph.ca', 'path' => '/hpc'),
```

Comment:
I think wellness.uoguelph.ca/hpc is no longer a separate Drupal site.

Reply:
Removed.